src/executor/LanceurExecutor.java

```diff
@@ -38,7 +38,7 @@ public Integer call() throws Exception {
                         }
 			
 		} catch (Exception e ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00108)